README.md

```diff
@@ -41,16 +44,8 @@ curl -L https://github.com/DataBiosphere/terra-cli/releases/latest/download/down
 This will install the Terra CLI in the current directory. Afterwards, you may want to add it to your `$PATH` directly
 or move it to a place that is already on your `$PATH` (e.g. `/usr/local/bin`).
 
-#### Manual install
-A Terra CLI release includes a GitHub release of the `terra-cli` repository and a corresponding Docker image in GCR.
-`download-install.sh` is a convenience script that downloads the latest (or specific version) of the install package,
-unarchives it, runs the `install.sh` script included inside, and then deletes the install package.
-
-You can also skip the `download-install.sh` script and install manually.
-- Download the `terra-cli.tar` install package directly from the 
-[GitHub releases page.](https://github.com/DataBiosphere/terra-cli/releases)
-- Unarchive the `tar` file.
-- Run the install script from the unarchived directory: `./install.sh`
+Re-installing will overwrite any existing installation (i.e. all JARs and scripts will be overwritten), but will not
+modify the `$PATH`. So if you have added it to your `$PATH`, that step needs to be repeated after each install.
```

Comment:
This is a little confusing to me.  Is this meant to address the case where the user has moved the install somewhere other than `~/.terra` ?

Reply:
The install script installs the Terra CLI in the current directory, but does not modify your `$PATH` (i.e. you have to call `./terra` instead of `terra`). This is a common install pattern, and specifically mimics the installer for Nextflow.

However, it also means that if you do want to add the Terra CLI to your path, you need to do so as a separate step after install. This note was just a reminder that neither installing nor re-installing affect the `$PATH`.

Reply:
Ah, I see.  So the libs/scripts get overwritten in `~/.terra` and the new exe lands in `pwd` (which may or may not overwrite the existing exe, depending on where the existing exe lives).  This is just a reminder to make sure `$PATH` resolves to the new exe (regardless of how it got onto the path).

Reply:
Yes, exactly.